lib/fog/azurerm/docs/storage.md

```diff
@@ -176,42 +166,119 @@ result = container.destroy
 puts "#{result}"
 ```
 
-### Metadata
+## Upload a local file as a blob
+```ruby
+new_blob = azure_storage_service.upload_block_blob_from_file('<container name>', '<blob name>', '<file path>')
+puts "#{new_blob.inspect}"
+```
+
+## Download a blob to a local file
+```ruby
+blob = azure_storage_service.download_blob_to_file('<container name>', '<blob name>', '<file path>')
+puts "#{blob.inspect}"
+puts "File Size: #{File.size <file_path>}"
+```
+
+## Delete the storage blob
+
+Mark the specified blob for deletion. The blob is later deleted during garbage collection.
+
+```ruby
+blob = azure_storage_service.blobs(container_name: '<container name>').get('<blob name>')
```

Comment:
We figured out that get request like service.collection(arg 1).get(arg 2) is not intuitive. In our recent development, we are using service.collection.get(arg1, arg2)

We will be updating code that references old way of getting objects. Therefore, I suggest that we follow new format of getting objects.